packages/bot-web-ui/src/pages/bot-builder/quick-strategy/form-wrappers/desktop-form-wrapper.tsx

```diff
@@ -27,6 +27,7 @@ type TDesktopFormWrapper = {
 
 const FormWrapper: React.FC<TDesktopFormWrapper> = observer(({ children, onClickClose, active_tab_ref }) => {
     const [activeTab, setActiveTab] = React.useState('TRADE_PARAMETERS');
+    const scroll_ref = React.useRef<HTMLDivElement & SVGSVGElement>(null);
```

Comment:
Do we need SVGSVGElement? Just HTMLDivElement should be fine right? @maryia-matskevich-deriv

Reply:
<ThemedScrollbars component expects refSetter prop with the type HTMLDivElement & SVGSVGElement

Reply:
alright